controllers/rcj_soccer_referee_supervisor/rcj_soccer_referee_supervisor.py

```diff
@@ -7,11 +7,14 @@
 from referee.referee import RCJSoccerReferee
 from recorder.recorder import VideoRecordAssistant
 
-automatic_mode = False
+from os import environ
 
-TEAM_YELLOW = "The Yellows"
-TEAM_BLUE = "The Blues"
+automatic_mode = True if "RCJ_SIM_AUTO_MODE" in environ.keys() else False
 
+TEAM_YELLOW = environ.get("TEAM_YELLOW_NAME", "The Yellows")
+TEAM_YELLOW_ID = environ.get("TEAM_YELLOW_ID")
+TEAM_BLUE = environ.get("TEAM_BLUE_NAME", "The Blues")
+TEAM_BLUE_ID = environ.get("TEAM_BLUE_ID")
```

Comment:
This should be environ.get("TEAM_BLUE_ID", 1), because otherwise we end up having None in the name of json file.